tensorflow_addons/rnn/cell.py

```diff
@@ -226,26 +231,27 @@ class LayerNormLSTMCell(keras.layers.LSTMCell):
     Stanislau Semeniuta, Aliaksei Severyn, Erhardt Barth.
     """
 
+    @typechecked
     def __init__(self,
-                 units,
-                 activation='tanh',
-                 recurrent_activation='sigmoid',
-                 use_bias=True,
-                 kernel_initializer='glorot_uniform',
-                 recurrent_initializer='orthogonal',
-                 bias_initializer='zeros',
-                 unit_forget_bias=True,
-                 kernel_regularizer=None,
-                 recurrent_regularizer=None,
-                 bias_regularizer=None,
-                 kernel_constraint=None,
-                 recurrent_constraint=None,
-                 bias_constraint=None,
-                 dropout=0.,
-                 recurrent_dropout=0.,
-                 norm_gamma_initializer='ones',
-                 norm_beta_initializer='zeros',
-                 norm_epsilon=1e-3,
+                 units: int,
+                 activation: str = 'tanh',
+                 recurrent_activation: str = 'sigmoid',
```

Comment:
They can be functions too (I'm not 100% sure on this one). Can you take a look at the code?

Reply:
Thanks @gabrieldemarmiesse for pointing out. Yes, there are functions.